src/queue.cpp

```diff
@@ -43,6 +44,16 @@ cvk_command_queue::cvk_command_queue(
     if (properties & CL_QUEUE_OUT_OF_ORDER_EXEC_MODE_ENABLE) {
         cvk_warn_fn("out-of-order execution enabled, will be ignored");
     }
+
+    TRACE_CNT_VAR_INIT(batch_in_flight_counter,
+                       "clvk-queue_" + std::to_string((uintptr_t)this) +
```

Comment:
I agree it would be better.
The idea here is to be able to easily link the counter with other events with the queue in argument. Unfortunately perfetto viewer prints integer in decimal format. So it would make it difficult to follow at the moment to put it here in hex.
I don't see any way of forcing perfetto to print integers in hex at the moment...

Reply:
Ah right. Agree that consistency is better.